build-image/Dockerfile

```diff
@@ -1,18 +0,0 @@
-from centos:7
```

Comment:
Seemed unnecessary to have this docker file since we can accomplish the same things with `golang` but I may have missed something so glad to undo this if needed.

Reply:
`build-image` has script for building the kube-router image from Mac. See #135

This is not hooked into current CI pipeline. So the changes are not related to unit test, so may be keep this PR just for unit test changes?